app/views/providers/capital_income_assessment_results/_bank_statements.html.erb

```diff
@@ -1,7 +1,8 @@
-<h3 class="govuk-heading-m"><%= t(".title") %></h3>
+<h3 class="govuk-heading-m"><%= t(".title", individual:) %></h3>
 <%= govuk_table do |table| %>
   <%= table.with_body do |body| %>
     <%= body.with_row do |row| %>
+      <p><%= individual %></p>
```

Comment:
Is this needed or just debug? 

We rarely seem to use just <p> tags

Reply:
Ah yes good catch, it is just a debug